src/main/java/core/basesyntax/impl/StorageImpl.java

```diff
@@ -3,17 +3,61 @@
 import core.basesyntax.Storage;
 
 public class StorageImpl<K, V> implements Storage<K, V> {
+    public static final int STARTING_CAPACITY = 10;
+    private K[] keys = (K[]) new Object[STARTING_CAPACITY];
```

Comment:
```suggestion
    private K[] keys;
```
we need do this in constuctor